framework/doc/content/sqa/framework_sdd.md

```diff
@@ -57,6 +57,88 @@ input file.
 
 !template-end!
 
+!template! item key=system-purpose
+The Software Design Description provided here is description of each object in the system. The pluggable
```

Comment:
I thought about this, but the "purpose" is covered in a lot of detail in the SRS document. I read the text in the INL template and went with what I wrote here because of that:

"...describing the system design in the user’s terminology, providing a guide for a more technical design document..."

"...Explain how this document might evolve throughout the product life cycle..."

So it's more of a description of the content of this document. I could be wrong, but I didn't want to repeat the content in the SRS.